src/contact.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Contact
+layout: "layouts/general-content.html"
+boxContent:
+  - If you have a question about plants that our databases can’t answer, don't despair! If you live in Minnesota, visit the University of Minnesota Extension's Gardening Information page. If you live anywhere in the USA you can ask a Master Gardener your question through the national eXtension system.
```

Comment:
All of these content items seem good, and are probably fine as is. Not sure if `eXtension` should be `extension`. If there are any links we can provide, that might be helpful? Otherwise this seems useful and can be updated more later.